src/Microsoft.AspNetCore.OData/Results/BadRequestODataResult.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.OData;
+
+namespace Microsoft.AspNet.OData.Results
+{
+    /// <summary>
+    /// Represents a result that when executed will produce a Bad Request (400) response.
+    /// </summary>
+    /// <remarks>This result creates an <see cref="ODataError"/> with status code: 400.</remarks>
+    public class BadRequestODataResult : BadRequestResult
+    {
+        /// <summary>
+        /// OData Error Message.
+        /// </summary>
+        public string Message { get; set; }
```

Comment:
>public string Message { get; set; } [](start = 8, length = 35)

public string Message => Error.Message;  or remove it

Reply:
Removed it.
#Resolved